justfile

```diff
@@ -76,7 +76,7 @@ build-docker-images:
     scripts/build-docker-images-native
 
 # generate rust bindings for contracts
-REGEXP := "^LightClient$|^LightClientStateUpdateVK$|^FeeContract$|PlonkVerifier$|^ERC1967Proxy$|^LightClientMock$|^LightClientStateUpdateVKMock$|^PlonkVerifier2$"
+REGEXP := "^LightClient$|^LightClientStateUpdateVK$|^FeeContract$|^HotShot$|PlonkVerifier$|^ERC1967Proxy$|^OwnableUpgradeable$|^LightClientMock$|^LightClientStateUpdateVKMock$|^PlonkVerifier2$"
```

Comment:
The HotShot contract has been removed.

Reply:
thanks, i missed that handling a merge conflict https://github.com/EspressoSystems/espresso-sequencer/commit/beeb04b27d4c69ff1148a84d67b36655f4ddc2f3